### ISSUE TITLE
get CircularList from FocusRing

### ISSUE THREAD

**Issue by 4eUeP** [jtdaugherty/brick#257]
I want to add `FocusRing` to instance of `ToJSON`:

```hs
instance (ToJSON n) => ToJSON (FocusRing n) where
  toJSON ring = ...
```

but cant get the internal `CircularList` by `FocusRing`, because the lack of exporting data constructor `FocusRing`. Should this be exported?

Or providing a function: `FocusRing n -> [n]` ?

**Comment by jtdaugherty** [comment]
Yes, I'd be okay with adding a new function `focusRingToList :: FocusRing n -> [n]`. That, combined with `focusGetCurrent`, `focusSetCurrent`, and `focusRing`, should provide enough of an API to be useful with `ToJSON` and `FromJSON`. How does that sound?

**Comment by 4eUeP** [comment]
Sounds good, I think I can implement this :) 
(will open a pull request later if you'd accept)

**Comment by jtdaugherty** [comment]
Great!

**Comment by jtdaugherty** [comment]
This change is now released in `0.51`.
